cupy_setup_build.py

```diff
@@ -520,6 +535,21 @@ def preconfigure_modules(compiler, settings):
             if os.path.exists(lib_path):
                 settings['library_dirs'].append(lib_path)
 
+        # In ROCm 4.1 and later, we need to use the independent version of
```

Comment:
For later record to myself: there are two hipfft header / library included, one is hipFFT included in rocFFT (`/opt/rocm-4.2.0/rocfft/include/hipfft.h`), which is deprecated in ROCm 4.1. We should instead use independent hipFFT (`/opt/rocm-4.2.0/hipfft/include/hipfft.h`).